src/fprime/util/commands.py

```diff
@@ -182,3 +184,53 @@ def run_code_format(
     for filename in parsed.files:
         clang_formatter.stage_file(Path(filename))
     return clang_formatter.execute(build, parsed.path, ({}, parsed.pass_through))
+
+
+def run_sysinfo(
+    base: Build, parsed: argparse.Namespace, _: Dict[str, str], __: Dict[str, str], ___
+):
+    """Print out versions to help debugging"""
+
+    print(f"Python version: {sys.version_info.major}{sys.version_info.minor}.{sys.version_info.micro}")
+
+    try:
+        import subprocess
+        cmake_version = subprocess.check_output(['cmake', '--version']).decode('utf-8').splitlines()[0].split()[2]
+        print(f"CMake version: {cmake_version}")
+    except ImportError: # Python >=3.6
+        print("[WARNING] Cannot import 'subprocess'. ")
+
+    try:
+        import pip
+        print(f"Pip version: {pip.__version__}")
+    except ModuleNotFoundError: # Python >=3.6
+        print("[WARNING] Cannot import 'Pip'.")
+
+    try:
+        import pkg_resources
+    except ModuleNotFoundError: # Python >=3.6
+        print("[WARNING] Cannot import 'pkg_resources'. Will not check tool versions.")
+        return
+
+    tools = [
```

Comment:
This list is the same as here: https://github.com/0x48piraj/fprime-tools/blob/863bde64a8965b881d1fdddd4729437e10d9d4b6/src/fprime/util/build_helper.py#L71-L85

Could we define this at the module level in build_helper.py, and import that variable here, instead of re-defining it?

Reply:
How about we take the variable from **build_helper.py** and put it over versioning.py inside a function called `get_toolchain` which returns the list?

Reply:
Yes perfect, I wouldn't put it in a function though, just FPRIME_PIP_PACKAGES global variable or something